project/frontend/src/components/ProfileForm.js

```diff
@@ -0,0 +1,69 @@
+import './ProfileForm.css';
+
+import React, { useState } from 'react';
+
+import CuisineAutocomplete from './CuisineAutocomplete.js';
+import ButtonGroup from './ButtonGroup.js';
+
+import Place from '../assets/place.svg';
+import Cuisine from '../assets/cuisine.svg';
+import Distance from '../assets/distance.svg';
+import Experience from '../assets/food_service.svg';
+import Price from '../assets/dollar.svg';
+
+/**
+ * A form used to gather the restaurant preferences of a user for the 'Find My Match'
+ * service.
+ *
+ * @param {!Object<string, number>} currLocation The current latitude and longitude
+ *     coordinates of the user.
+ * @param {string} locationName The formatted address of the current user.
+ */
+function ProfileForm(props) {
+  const cuisineOptions = ['Italian', 'Mexican', 'Indian'];
+  const [cuisine, setCuisine] = useState([]);
+
+  return (
+    <div className='profile-form-container'>
+      <form className='profile-form'>
+        <h4 className='profile-header'>Your profile.</h4>
+        <div className='profile-form-row'>
+          <img src={Place} alt='' />
+          <label>Your Location</label>
+          {/* TODO: change so that if a location isn't inputted then we are going to need to have a fill in optiob. */}
+          <p>Your location</p>
+        </div>
+        <div className='profile-form-row'>
+          <img src={Cuisine} alt='' />
+          <label htmlFor='cuisine'>Preferred Cuisines</label>
```

Comment:
We use the htmlFor to associate a label with an html element. So whatever this is labeling needs to have `id='cuisine'` so that you know what it is to be associated with and same with all the ones below (distance, price, experience). Like in this case, I think we'd want to make 'cuisine' the id for the CuisineAutocomplete component. The responses here explain this nicely: https://stackoverflow.com/questions/698001/whats-the-for-for-in-a-label-tag

Reply:
gotcha! should i also change this in the preference form then? I was just following that structure

Reply:
We can open a new PR to make those changes on PreferenceForm (adding alt text, fixing labels) :)

Reply:
Also regarding labels, I think we talked about this way earlier but instead of using for you can also wrap the thing that is being labeled in the label tag if you want.. I'm pretty sure you had used it that way for something earlier, so that would work as well since it still binds the label to the element (under "Tips and Notes") https://www.w3schools.com/tags/tag_label.asp